### ISSUE TITLE
multiple hyphens

### ISSUE THREAD

**Issue by auguszou** [simov/slugify#82]
```
slugify("developer @ Google", {
        lower: true,
        strict: true
      })
```
will return `developer--google`

still will cuase multiple hyphens

**Comment by auguszou** [comment]
and `@` should be converted to `at`

**Comment by Trott** [comment]
I said this in another issue several minutes ago, but I hope it's not bad form to recommend another package in the issue tracker when an issue has been open for a while.

The `slug` package and the `@sindresorhus/slugify` package will both de-duplicate the hyphens by default, returning `'developer-google'`.

(Full disclosure: I'm the current maintainer of `slug`.)

**Comment by simov** [comment]
Fixed in v1.4.1
